From 19c8d8517e3d9b23fb68060df667d03189b890f9 Mon Sep 17 00:00:00 2001
From: Markos Chandras <markos.chandras@imgtec.com>
Date: Mon, 17 Jun 2013 07:53:47 +0000
Subject: [PATCH 016/105] MIPS: powertv: Drop SYS_HAS_EARLY_PRINTK

PowerTV does not provide a prom_putchar function needed for early
printk so remove this symbol for this platform.

Fixes the following problem when EARLY_PRINTK is enabled for powertv:

arch/mips/kernel/early_printk.c:24: undefined reference to `prom_putchar'
arch/mips/kernel/early_printk.c:23: undefined reference to `prom_putchar'

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
Acked-by: Steven J. Hill <Steven.Hill@imgtec.com>
Cc: linux-mips@linux-mips.org
Patchwork: https://patchwork.linux-mips.org/patch/5477/
Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
(cherry picked from commit c37441c127e000869a960a866fe2207626935e4f)
---
 arch/mips/Kconfig |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index e53e2b4..97dd298 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -420,7 +420,6 @@ config POWERTV
 	select CSRC_POWERTV
 	select DMA_NONCOHERENT
 	select HW_HAS_PCI
-	select SYS_HAS_EARLY_PRINTK
 	select SYS_HAS_CPU_MIPS32_R2
 	select SYS_SUPPORTS_32BIT_KERNEL
 	select SYS_SUPPORTS_BIG_ENDIAN
-- 
1.7.1

